evennia/contrib/base_systems/godotwebsocket/webclient.py

```diff
@@ -0,0 +1,78 @@
+"""
+Godot Websocket - ChrisLR 2022
+
+This file contains the code necessary to dedicate a port to communicate with Godot via Websockets.
```

Comment:
For clarity, it may be worth mentioning that this is part of the plugin system and that you plug it in from the settings described in the README.